term/functions.php

```diff
@@ -34,6 +34,29 @@ function term_block_styles() {
 				}',
 			)
 		);
+
+		register_block_style(
+			'core/group',
+			array(
+				'name' => 'hover-effect',
+				'label' => __( 'Hover effect', 'term'),
+				'inline_style' => '
+				.is-style-hover-effect {
+					background-color: #EDECE8;
```

Comment:
this gives a really nice effect but it has other side effects, namely:

- If the user decides to change the original color palette (or if we decide to add additional styles, which is likely), the hover effect may no longer match the new colors
- If the user adds new elements/blocks like a post date or similar, we no longer cover that possibility.

Reply:
Yes it's true, it should be a preset color